packages/experiments/src/components/KeytipLayer/examples/KeytipLayer.Basic.Example.tsx

```diff
@@ -0,0 +1,237 @@
+                text='Test Button 3'
+                onClick={ () => { alert('Button 3'); } }
+              />
+            </div>
+          </div>
+          <div style={ divStyle }>
+            <div>
+              <ActionButton
+                data-ktp-id={ convertSequencesToKeytipID(this.keytipMap.Pivot2Keytip.keySequences) }
+                text='Mock Pivot 2'
+              />
+            </div>
+            <CommandBar
+              items={
+                [
+                  {
+                    key: 'commandBarItem1',
+                    name: 'New',
+                    icon: 'Add',
+                    onClick: this._showModal,
+                    ['data-ktp-id']: convertSequencesToKeytipID(this.keytipMap.CommandButton1Pivot2Keytip.keySequences)
+                  },
+                  {
+                    key: 'commandBarItem2',
+                    name: 'Upload',
+                    icon: 'Upload',
+                    onClick: this._showMessageBar,
+                    ['data-ktp-id']: convertSequencesToKeytipID(this.keytipMap.CommandButton2Pivot2Keytip.keySequences)
+                  }
+                ]
+              }
+              farItems={
+                [
+                  {
+                    key: 'farItem1',
+                    name: 'SubMenu',
+                    icon: 'SortLines',
+                    ['data-ktp-id']: convertSequencesToKeytipID(this.keytipMap.CommandButton3Pivot2Keytip.keySequences),
+                    subMenuProps: {
+                      items: [
+                        {
+                          key: 'emailMessage',
+                          name: 'Email message',
+                          icon: 'Mail',
+                        },
+                        {
+                          key: 'calendarEvent',
+                          name: 'Calendar event',
+                          icon: 'Calendar'
+                        }
+                      ],
+                    },
+                  }
+                ]
+              }
+            />
+          </div>
+        </div>
+        { this.state.showMessageBar &&
+          <MessageBar messageBarType={ MessageBarType.success }>
+            Success Uploading
+          </MessageBar>
+        }
+        <Modal
+          isOpen={ this.state.showModal }
+          onDismiss={ this._hideModal }
+          isBlocking={ false }
+        >
+          <h2>Hello this is a Modal</h2>
+        </Modal>
+        <KeytipLayer
+          keytipStartSequences={ [this.startingKeySequence] }
+          keytipExitSequences={ [this.startingKeySequence] }
+          id={ 'test-id' }
+        />
+      </div>
+    );
+  }
+
+  public componentDidMount(): void {
+    // Manually add keytips to the KeytipManager for now
+    // This should really be done in each component
+    for (let component of Object.keys(this.keytipMap)) {
+      registerKeytip(this.keytipMap[component]);
```

Comment:
I see a `registerKeytip` call but no `unregisterKeytip`.

I am really nervous about this pattern because if it's easy for you to forget to do this, it will be easy for partners to as well.

Reply:
Yes, this is also mocked, the register and unregister should only be in the component render() and componentWillUnmount() (I know we have discussed this, we can try to come up with a better solution). In this sample we don't have any dynamic DOM elements so we aren't doing any unregistering. When we move to the normal package we will have keytips on menus, etc

Reply:
isn't the sample itself dynamic? E.g. you will have a memory leak if you go to the page, then to another, then back to the original.